dockers/texcmp/README.md

```diff
@@ -0,0 +1,73 @@
+# How to compare against LaTeX
+
+The tools in this directory can be used to create reference images
+using LaTeX, and to compare them against the screenshots taken from a
+browser.
+
+## Execution environment
+
+### Docker environment
+
+If you don't want to ensure the presence of all required tools, or
+want to make sure that you create reproducible results, simply run
+
+    dockers/texcmp/texcmp.sh
+
+from the root of your KaTeX directory tree.
+This will build a suitable docker image unless such an image already
+exists.  It will then use a container based on that image to generate
+all the images described below.
+
+Note that the files and directories created inside the docker will be
+owned by root, so you might have trouble deleting them later on.  Be
+sure you can obtain superuser permissions on your computer or know
+someone who can, just to be safe.
```

Comment:
Right. I wonder whether we should be streaming out those files via a TCP connection instead of a `--volume` mount. Or pre-create the files outside. Or anything else like this. But I see that as a separate pull request, once the core functionality is in place.